docs/PLUGIN_CATALOG.md

```diff
@@ -10,21 +10,22 @@ This is the master catalog of plugins for Snap. While the Snap framework is hard
 ## All Plugins
 This file is automatically generated. If you would like to add to the plugin list, [add your plugin to this list](plugins.yml) and it will be added (usually within 24 hours).
 
-### Collectors (61)
+### Collectors (62)
 
 | Name | Maintainer | Description | CI | Download |
 |------|------------|-------------|----|----------|
-| [Apache](https://github.com/intelsdi-x/snap-plugin-collector-apache) | [Intel](https://github.com/intelsdi-x) | Collects Apache webserver mod_status metrics | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-apache.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-apache) [![Build Status](https://ci.snap-telemetry.io/job/snap-plugin-collector-apache-ec2-periodic/badge/icon)](https://ci.snap-telemetry.io/job/snap-plugin-collector-apache-ec2-periodic/) | [release](https://github.com/intelsdi-x/snap-plugin-collector-apache/releases) |
```

Comment:
So I traced this one - running previous versions of plugin sync does not bring back the large test banners. Something has changed in each repo's metadata, probably due to plugin sync itself. Still tracing it.